packages/config/config/devices/0x0234/templates/logic_group_template.json

```
@@ -0,0 +1,550 @@
+				"value": 4
+			},
+			{
+				"label": "Cooling meter.",
+				"value": 5
+			}
+		]
+	},
+	"input_scale": {
+		"label": "Scale Type of input",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 6,
+		"defaultValue": 3,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"input_precision": {
+		"label": "Precision of input",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 7,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"pulse_unit": {
+		"label": "Pulses per Unit of input 5",
+		"unit": "pulse",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 65000,
+		"defaultValue": 1,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"rate_type": {
+		"label": "Rate Type of input",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 2,
+		"defaultValue": 1,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Unspecified",
+				"value": 0
+			},
+			{
+				"label": "Import",
+				"value": 1
+			},
+			{
+				"label": "Export",
+				"value": 2
+			}
+		]
+	},
+	"time_interval_report": {
+		"label": "Time interval between meter reports",
+		"unit": "100 ms",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 8640,
+		"defaultValue": 90,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"led_brightness": {
+		"$import": "../../templates/master_template.json#base_0-99_nounit",
+		"label": "Brightness of Status LED",
+		"defaultValue": 50
+	},
+	"input_calibration": {
+		"$import": "../../templates/master_template.json#base_0-99_nounit",
+		"label": "Input Calibration",
+		"unit": "0.1 °C",
+		"minValue": -40,
+		"maxValue": 40,
+		"defaultValue": 0,
+		"unsigned": false
+	},
+	"input_report_interval": {
+		"label": "Input Report Interval",
+		"unit": "10 seconds",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 8640,
+		"defaultValue": 6,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"input_function_setup": {
+		"label": "Function setup of Input",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 6,
+		"defaultValue": 3,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true,
+		"options": [
+			{
+				"label": "Level-controlled, NO",
+				"value": 0
+			},
+			{
+				"label": "Level-controlled, NC",
+				"value": 1
+			},
+			{
+				"label": "Toggle-controlled, NO",
+				"value": 2
+			},
+			{
+				"label": "Toggle-controlled, NC",
+				"value": 3
+			},
+			{
+				"label": "Timer-controlled, NO",
+				"value": 4
+			},
+			{
+				"label": "Timer-controlled, NC",
+				"value": 5
+			}
+		]
+	},
+	"pushbutton_debounce": {
+		"$import": "../../templates/master_template.json#base_1-255_nounit",
+		"label": "Pushbutton debounce timer",
```

This is another case of "check the style guide":
> Labels should be **Title Case**.
```suggestion
		"label": "Pushbutton Debounce Timer",
```